### DISCUSSION TITLE
Account ID

### DISCUSSION THREAD

**Discussion by lmchp** [maheini/FreeStyle-Libre-3-patch#48] [Q&A]
Bei der Libre-3 Installation inkl. LibreView Account Einrichtung hatte sich einiges verhakelt, so dass die Juggluco-Übernahme scheiterte... nun 14 Tage später neuer Versuch, aber:
...ich bekomme im Juggluco per poll eine plausible Account ID via LibreView,
jedoch wird diese dann beim highjacking (=NFC-scan) des Sensors als FALSCH bemängelt!?
(siehe: https://github.com/maheini/FreeStyle-Libre-3-patch/issues/47)

Q: kennt dazu jemand eine Lösung?

Anm.: j-kaltes hatte angemerkt, dass sich die ID auch mit seinem  Java-Konverter
 (https://www.juggluco.nl/Juggluco/libre3/nolibreview.html) aus einer UID, die jedoch
 nur per root-Zugriff aus der Libre-3 Installation zu bekommen sei generieren ließe.
Auf meinem gerooteten 2. Gerät (nicht Magisk) hatte ich Libre 3 nicht installiert bekommen...
habe aber trotzdem mit dem  Java-Konverter etwas rumgespielt - und siehe da:

Reports in LibreView enthalten in der URL eine UID in Form:
https://www.libreview.com/reports/[UID]
Der Java-Konverter erzeugt bei mir aus DIESER UID exakt die (s.o.: falsche) Account-ID, die Juggluco
für mich ermittelt hat.

1. Kann jemand mit funktionierendem Setup das bitte mal testen und mitteilen, ob sich die
zum Sensor passende Account ID per Konverter aus der Report-UID errechnen lässt?
2. Falls ja, dann auch zur Info an j-kaltes, dass root-Zugriff nich notwendig ist....
3. Dank für Jeden weiteren Tipp / Anregung, wie ich zu funktionierender Account-ID komme ;-)

**Comment by maheini** [comment]
Hmm... komisch. Ich habe im Moment nicht viel Zeit, werde mir den Bericht später in Ruhe durchlesen. Folgendes Setup funktioniert allerdings definitiv: Juggluco 4.11.2, Android 12, kein Root, Huawei Gerät. Libre kann aktiviert werden und auch übernommen werden.
